server/models/updates/synt.sql

```diff
@@ -373,3 +373,11 @@ ALTER TABLE account DROP COLUMN is_ohada;
 ALTER TABLE account DROP FOREIGN KEY account_ibfk_1;
 ALTER TABLE account_type MODIFY id mediumint(8) UNSIGNED NOT NULL AUTO_INCREMENT;
 ALTER TABLE account ADD FOREIGN KEY (`account_type_id`) REFERENCES account_type (`id`);
+
+-- Add the property is locked for the projet 
+-- Date: 2016-01-28
+-- By: Chris LOMAME
+
+ALTER TABLE `project`
```

Comment:
Most of our tables call this property `locked`.  You don't need the `is_` prefix.